src/Components/ProductOverview/ProductOverview.jsx

```diff
@@ -1,23 +1,21 @@
 import React from 'react';
 import ProductInfo from './ProductInfo/ProductInfo';
+import DescriptionFeaturesShare from './DescriptionFeaturesShare/DescriptionFeaturesShare';
+import './ProductOverview.css';
 
 const ProductOverview = () => (
   <div className="product-overview">
     <div className="container-fluid">
       <div className="row">
-        <span className="border border-secondary">
-          <div className="col-12 col-md-8 image-gallery">ImageGallery</div>
-        </span>
-        <span className="border border-secondary">
-          <div className="col-12 col-md-4 product-info">
-            <ProductInfo />
-          </div>
-        </span>
+        <div className="border col-12 col-sm-8 col-lg-8 col-xl-8 image-gallery">
+          ImageGallery
+        </div>
+        <div className="border col-12 col-sm-4 col-lg-4 col-xl-4 product-info">
+          <ProductInfo />
+        </div>
       </div>
-      <div className="row">
-        <span className="border border-secondary">
-          <div className="col-12">Description, Features, and Share</div>
-        </span>
```

Comment:
These <span> border border-secondary were creating errors for my bootstrap grid, thus removed.